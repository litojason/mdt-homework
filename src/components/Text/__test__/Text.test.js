import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import { Text } from "../index";

describe("Component: Text", () => {
  it("should render Text", () => {
    const component = <Text />;

    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
