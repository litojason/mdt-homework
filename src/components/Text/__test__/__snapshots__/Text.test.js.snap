// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Text should render Text 1`] = `
.c0 {
  font-size: 1rem;
  font-weight: normal;
}

<div
  className="c0"
/>
`;
